src/main/java/org/euvsvirus/cac/controller/UserController.java

```diff
@@ -35,9 +46,30 @@ public User me() {
         return null;
     }
 
-    @PostMapping(value = "/login", produces = MediaType.APPLICATION_JSON_VALUE, consumes = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<JWTTokenResponse> login(@RequestBody LoginRequest login) {
+    @RequestMapping(value = "/login", method = RequestMethod.POST)
+    public ResponseEntity<?> createAuthenticationToken(@RequestBody LoginRequest loginRequest) throws Exception {
+        authenticate(loginRequest.getEmail(), loginRequest.getPassword());
+        final UserDetails userDetails = cacUserDetailsService.loadUserByUsername(loginRequest.getEmail());
+        final String token = jwtTokenService.generateToken(userDetails);
+        return ResponseEntity.ok(new JWTTokenResponse(token));
+    }
 
-        return new ResponseEntity<>(cacUserLoginService.login(login.getEmail(), login.getPassword()), HttpStatus.OK);
+    @PostMapping("/user")
```

Comment:
Würde ich `/register` oder `/signup` nennen, weil sonst sieht das in der SecurityConfig so aus, aus würde man an sensible Dinge rankommen.